openedx/core/djangoapps/user_authn/views/tests/test_auth.py

```diff
@@ -0,0 +1,22 @@
+from django.test import TestCase, Client
+from django.conf import settings
+from common.djangoapps.util.json_request import JsonResponse
+from unittest.mock import patch
+
+class getPublicSigningJWKSFunctionTest(TestCase):
+    def setUp(self):
+        self.client = Client()
+
+    @patch('common.djangoapps.util.json_request.JsonResponse')
+    def test_get_public_signing_jwks_with_no_jwk_set(self, mock_json_response):
+        settings.JWT_AUTH = {}  
```

Comment:
See https://docs.djangoproject.com/en/4.2/topics/testing/tools/#overriding-settings for implementation options.